docker/build-tools/prow-entrypoint.sh

```diff
@@ -35,4 +35,9 @@ if [[ -n "${GOOGLE_APPLICATION_CREDENTIALS:-}" ]]; then
   gcloud auth configure-docker -q || true
 fi
 
-exec "$@"
+"$@"
+EXIT_VALUE=$?
+
+docker system prune -f || true
```

Comment:
currently using `docker ps -aq | xargs -r docker rm -f || true` in the new image FWIW, I forget what prune covers